src/component/base/DraftEditor.react.js

```diff
@@ -280,6 +281,9 @@ class DraftEditor
             <DraftEditorContents
               blockRendererFn={nullthrows(this.props.blockRendererFn)}
               blockStyleFn={nullthrows(this.props.blockStyleFn)}
+              customBlockMap={
+                {...this.props.customBlockMap}
```

Comment:
As discussed, we treat customBlocks as a direct overwrite as opposed to extend, we should do the same for inlineStyles when we make them configurable